src/exportPdfCommand.ts

```diff
@@ -35,25 +35,27 @@ export class ExportPDFCommand extends UICommand {
     return false;
   };
 
-  cancel(): void {
+  public cancel(): void {
     return null;
   }
 
-  execute = (
+  public execute = (
     _state: EditorState,
     _dispatch: (tr: Transform) => void,
     view: EditorView
   ): boolean => {
     return this.exportPdf.exportPdf(view);
   };
 
-  renderLabel() {
+  public renderLabel(): null {
```

Comment:
return type should match parent's

Reply:
What do you mean? Its public method that only returns null, I just added the return type.